src/resources/views/components/summary-card.blade.php

```diff
@@ -3,13 +3,14 @@
     'list',
     'noBorders' => false,
     'title',
+    'id' => ''
 ])
 
 @php
     
 @endphp
 
-<div class="govuk-summary-card">
+<div class="govuk-summary-card" @if(!empty($id)) id="{{ $id }}" @endif>
```

Comment:
Use a proper comparator, please. `empty($id) === false`. I will not accept `!empty()` as while it is valid code, it is easy to miss / misread.